web/handlers/admin/invites_test.go

```diff
@@ -64,6 +64,50 @@ func TestInvitesOverview(t *testing.T) {
 	a.True(yes, "a-tag has href attribute")
 	wantURL := ts.URLTo(router.AdminInvitesRevokeConfirm, "id", 666)
 	a.Equal(wantURL.String(), link)
+
+	testInviteButtonDisabled := func(shouldBeDisabled bool) {
```

Comment:
great pattern with these funcs. Makes it very clear what is being tested below. :ok_hand: